tomviz/operators/OperatorPython.cxx

```diff
@@ -257,6 +257,14 @@ void OperatorPython::setJSONDescription(const QString& str)
     }
   }
 
+  // Get the number of parameters
+  QJsonValueRef parametersNode = root["parameters"];
+  if (!parametersNode.isUndefined() && !parametersNode.isNull()) {
+    QJsonArray parametersArray = parametersNode.toArray();
+    QJsonObject::size_type numParameters = parametersArray.size();
+    setNumberOfParameters(numParameters);
```

Comment:
What happens if the operator doesn't have JSON?  Delete Slices for example has arguments, but no JSON description.

Reply:
Please test this with https://github.com/OpenChemistry/tomviz/pull/1663 (you may need that to make Delete Slices work right).

Reply:
The EditOperatorDialog wouldn't be display on first run, but that was the case before as well, since the check was was whether or not a json description was provided.

The new numberOfParameters() method exists alongside the previous arguments() method, so I don't think it should break anything